### PR TITLE
dx-825 vipps redirect in steps

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: payments/vipps/redirect.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ that the payer must confirm through the Vipps mobile app." %}
 ## Introduction
 
 * When the payer starts the purchase process, you make a `POST` request towards
-  Swedbank Pay with the collected `Purchase` information.
-* This will generate a payment object with a unique `paymentID`.
+  Swedbank Pay with the collected `Purchase` information. This will generate a
+  payment object with a unique `paymentID`.
```
Comment (asbjornu) [inline payments/vipps/redirect.md]:
```suggestion
  payment with a unique `id`.
```

File: payments/vipps/redirect.md [hunk 2]
```diff
@@ -39,94 +39,16 @@ that the payer must confirm through the Vipps mobile app." %}
 * To receive the state of the transaction you need to do a `GET`
   request containing the `paymentID` generated in the first step.
```
Comment (asbjornu) [inline payments/vipps/redirect.md]:
```suggestion
  request containing the `id` of the payment generated in the first step.
```

File: payments/vipps/redirect.md [hunk 3]
```diff
@@ -286,6 +208,209 @@ Once the payment is successfully authorized by the payer, she is returned to
 On that page as well as in the `callbackUrl`, you need to perform an HTTP `GET`
 request towards the `id` of the payment to inspect its status.
 
+## Step 3: Get the transaction state
+
+The `GET`request below will give you the transaction state of the payment.
+The `paymentId` used below was provided in the fist step when creating a
```
Comment (asbjornu) [inline payments/vipps/redirect.md]:
```suggestion
The `id` of the payment used below was provided in the first step when creating a
```

File: payments/vipps/redirect.md [hunk 4]
```diff
@@ -286,6 +208,209 @@ Once the payment is successfully authorized by the payer, she is returned to
...
+        Merchant-->>SwedbankPay: GET <payment.id>
+        note left of Merchant: Second API request
+        SwedbankPay-->>Merchant: Payment resource
+        Merchant-->>Browser: Display authorize result
+      deactivate SwedbankPay
+    deactivate Merchant
+  deactivate Browser
+```
+
+### Intent
```
Comment (asbjornu) [inline payments/vipps/redirect.md]: I don't think `Intent` is suitable as a header here?

File: payments/vipps/redirect.md [hunk 5]
```diff
@@ -286,6 +208,209 @@ Once the payment is successfully authorized by the payer, she is returned to
...
+
+The `GET`request below will give you the transaction state of the payment.
+The `paymentId` used below was provided in the fist step when creating a
+purchase.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/{{ payment_instrument }}/payments/{{ page.payment_id }}/ HTTP/1.1
```
Comment (asbjornu) [inline payments/vipps/redirect.md]: I don't think `payment_instrument` will have a value here.

```suggestion
GET /psp/vipps/payments/{{ page.payment_id }}/ HTTP/1.1
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207&bid=17103202)

File: payments/vipps/index.md [hunk 6]
```diff
@@ -43,6 +43,16 @@ the merchant's site.
...
 
+### Intent
+
+**`Authorization` (two-phase)**: The intent of the payment identifies how and
+when the charge will be effectuated. This determines the type of transaction
+used during the payment process. The intent of a Vipps purchase is always
+`Authorization`. The amount will be reserved but not charged.
+You will later (i.e. if a physical product, when you are ready to ship the
+purchased products) have to make a [Capture][capture] or
+[Cancel][cancel] request.
```
Comment (payex-dev) [inline payments/vipps/index.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207)

File: payments/vipps/index.md [hunk 7]
```diff
@@ -43,6 +43,16 @@ the merchant's site.
...
 ## Good To Know
 
+### Intent
+
+**`Authorization` (two-phase)**: The intent of the payment identifies how and
+when the charge will be effectuated. This determines the type of transaction
+used during the payment process. The intent of a Vipps purchase is always
+`Authorization`. The amount will be reserved but not charged.
+You will later (i.e. if a physical product, when you are ready to ship the
+purchased products) have to make a [Capture][capture] or
```
Comment (payex-dev) [inline payments/vipps/index.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207&bid=17103202)

File: payments/vipps/index.md (hunk 6 above)
Comment (payex-dev) [inline payments/vipps/index.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207)

Comment (payex-dev) [inline payments/vipps/index.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207&bid=17103202)

File: payments/vipps/index.md (hunk 7 above)
Comment (payex-dev) [inline payments/vipps/index.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5216207)

**Pull request merged**